### ISSUE TITLE
Question: Punkt Sentence Tokenizer/Segmentation

### ISSUE THREAD

**Issue by pthieu** [NaturalNode/natural#294]
Read your blog post here: https://dzone.com/articles/using-natural-nlp-module

Wondering if `Punkt sentence segmentation` as been added? I don't see in README. If not, I can take a crack at it.

**Comment by Hugo-ter-Doest** [comment]
Yes, we are interested in such functionality. Maybe you can take a look at the regexp tokenizer and build from there. 

Hugo
